ichnaea/webapp/app.py

```diff
@@ -38,7 +44,26 @@ def worker_exit(server, worker):
     shutdown_worker(_APP)
 
 
+def log_access_factory(wsgi_app):
```

Comment:
This is a WSGI middleware that wraps the ichnaea webapp wsgi app. It logs data about the request and response. We can add other things to it if we like.